tools/ci_build/github/azure-pipelines/packaging-pipeline.yml

```diff
@@ -52,37 +75,54 @@ stages:
         inputs:
           ArtifactName: torch-ort
 
-      - script: |
-          sudo apt-get update
-          sudo apt-get install python3-pip python-dev
-        condition: succeededOrFailed()
-        displayName: 'sudo apt-get install python3-pip python-dev'
+      # now we have wheel from manylinux, test it with public dockers
+      - task: CmdLine@2
+        inputs:
+          script: |
+            docker build -f $(PublicDockerFile) -t torchortpackagingtest .
+        displayName: "docker build -f $(PublicDockerFile) -t torchortpackagingtest"
+
+      # dockerfile has torch-ort. uninstall it to prepare for new install from current build
+      - task: CmdLine@2
+        inputs:
+          script: |
+            docker run --rm \
+              torchortpackagingtest \
+                /usr/bin/python3 -m pip uninstall -y torch-ort
```

Comment:
I'm confused. Won't the docker container be torn down after this uninstall command runs? That makes it useless, no?

Reply:
good catch - removed --rm flag

Reply:
I don't think that's sufficient. The "--rm" flag just ensures the container doesn't leave a dead container around after the command completes. In the subsequent command, you'd have to re-start the dead container. Better yet, why not run the uninstall right before you plan to do the install Then both run in the same container.

Reply:
Sorry @suffiank I misunderstood you and thanks also to @baijumeswani for telling me the same. I have removed that ineffective step. when torch-ort is installed it uses --upgrade to make sure the current wheel is used.